aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -511,6 +509,22 @@ def resolve(self, event, context) -> Dict[str, Any]:
     def __call__(self, event, context) -> Any:
         return self.resolve(event, context)
 
+    @staticmethod
+    def _has_debug(debug: Optional[bool] = None) -> bool:
+        # It might have been explicitly switched off (debug=False)
+        if debug is not None:
+            return debug
+
+        # Maintenance: deprecate EVENT_HANDLER_DEBUG later in V2.
+        env_debug = os.getenv(constants.EVENT_HANDLER_DEBUG_ENV)
+        if env_debug is not None:
+            warnings.warn(
+                "POWERTOOLS_EVENT_HANDLER_DEBUG is set and will be deprecated in V2. Please use POWERTOOLS_DEV instead."
```

Comment:
We need to update the [documentation](https://awslabs.github.io/aws-lambda-powertools-python/1.29.2/core/event_handler/api_gateway/#debug-mode).

We have 2 options and in my opinion we should replace the environment variable name in the documentation.

Adding a warning
![image](https://user-images.githubusercontent.com/4295173/193946745-7641b7bc-e6eb-411d-8f7d-17c884d39ec8.png)

Replacing
![image](https://user-images.githubusercontent.com/4295173/193946900-96add6b1-61b3-4c34-90e5-d512ced7b6cd.png)

Reply:
Good catch, let's replace it. I think you meant to make a comment in the [docs PR](https://github.com/awslabs/aws-lambda-powertools-python/pull/1569) - I'll assume you approve the docs PR after this change, then I'll merge.

Reply:
done - resolving.